### DISCUSSION TITLE
Questions for VSCode codebase / development

### DISCUSSION THREAD

**Discussion by progrium** [microsoft/vscode-discussions#2391] [Extension Development QnA] [answered]
What is the best place to ask questions about the VSCode codebase and build process? I don't want to pollute issues if possible, but if that's the best option let me know.

**Comment by robconery** [comment]
I would suggest you start here: https://github.com/microsoft/vscode-discussions. The only discussions forum is this one, which is scoped to extensions. If you had a question, you could search the issues to see if there's an answer. Also: if you're a Copilot subscriber, you could ask `@vscode` a question in chat!

**Reply by progrium** [reply]
You're suggesting I post here anyway if I can't otherwise find an answer?

**Reply by robconery** [reply, marked as answer]
Oops that must have been confusing, sorry about that! I meant to have a look at the start page (https://github.com/microsoft/vscode-discussions/blob/main/README.md) and see if there is a good place for your question. We're trying to keep this repo focused on extensions.
